modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts

```diff
@@ -111,6 +111,25 @@ export async function parseI3STileContent(
   return await parseI3SNodeGeometry(arrayBuffer, content, tileOptions, tilesetOptions, options);
 }
 
+/**
+ * Get the URL inside SLPK archive
+ * @param url - full url with *.slpk prefix
+ * @returns URL inside SLPK archive
+ */
+function getInternalUrl(url: string) {
```

Comment:
- Nit: specify return type `string
-Just thinking: Is there a better (more self documenting) name than `getInternalUrl`? 
- maybe `getArchivePathfromUrl()`, `getInternalPathFromUrl()`...